reposerver/repository/repository.go

```diff
@@ -216,6 +216,10 @@ func GenerateManifests(root, path string, q *apiclient.ManifestRequest) (*apicli
 
 	appSourceType, err := GetAppSourceType(q.ApplicationSource, appPath)
 	creds := argo.GetRepoCreds(q.Repo)
+	repoURL := ""
```

Comment:
how come this is empty string by default?

Reply:
It can be blank, which is basically a NOP further downstream. I noticed that `q.Repo` might be nil under circumstances, and then we just pass in a blank down.